src/app/ui/bridge/BridgeExplanation.tsx

```diff
@@ -1,10 +1,10 @@
 'use client';
-import { BridgePageContainer } from '@/app/ui/bridge/BridgePage.style';
-import { Link as MuiLink, Typography } from '@mui/material';
+import { Typography } from '@mui/material';
+import { SeoPageContainer } from 'src/components/SeoPageContainer.style';
 
 const BridgeExplanationSection = () => {
   return (
-    <BridgePageContainer>
+    <SeoPageContainer>
```

Comment:
pls don't call it SEO

Reply:
call is TokenSwap or something like this

Reply:
I called it "DynamicPagesContainer" as this is also being used on other pages for the bridges, not just swap related.